RetroBar/Languages/català.xaml

```diff
@@ -19,24 +20,84 @@
+    <s:String x:Key="quick_launch_folder">Inici ràpid: trieu una carpeta</s:String>
+    <s:String x:Key="show_badges">Mostra els _distintius dels icones</s:String>
+    <s:String x:Key="show_window_previews">Mostra les mi_niatures de les finestres</s:String>
     <s:String x:Key="use_software_rendering">Utilitza _processament per programari</s:String>
+    <s:String x:Key="add_show_desktop_button">Afegeix el botó de Mostra l'_escriptori</s:String>
+    <s:String x:Key="enable_auto_hide">A_maga automàticament la barra de tasques</s:String>
+    <s:String x:Key="lock_taskbar">_Bloca la barra de tasques</s:String>
+    <s:String x:Key="multiple_displays">Diverses pantalles</s:String>
+    <s:String x:Key="show_tasks_on">Mostra les _tasques en:</s:String>
+    <x:Array x:Key="show_tasks_on_values" Type="s:String">
+        <s:String>Totes les barres de tasques</s:String>
+        <s:String>La mateixa barra de tasques que la finestra</s:String>
+        <s:String>La mateixa barra de tasques que la finestra i la barra de tasques principal</s:String>
+    </x:Array>
+    <s:String x:Key="invert_system_icons">_Inverteix les icones del sistema:</s:String>
+    <x:Array x:Key="invert_system_icons_values" Type="s:String">
+        <s:String>Quan el tema ho requereixi</s:String>
+        <s:String>Sempre</s:String>
+        <s:String>Mai</s:String>
+    </x:Array>
+    <s:String x:Key="middle_mouse_task_action">Acció del botó _central:</s:String>
+    <x:Array x:Key="middle_mouse_task_action_values" Type="s:String">
+        <s:String>No facis res</s:String>
+        <s:String>Obre una nova instància</s:String>
+        <s:String>Tanca la tasca</s:String>
+    </x:Array>
+    <s:String x:Key="clock_click_action">Acció de _fer clic al rellotge:</s:String>
+    <x:Array x:Key="clock_click_action_values" Type="s:String">
+        <s:String>No facis res</s:String>
+        <s:String>Obre el calendari Aero</s:String>
+        <s:String>Obre el calendari Modern</s:String>
+        <s:String>Obre el centre de notificacions</s:String>
+    </x:Array>
+    <s:String x:Key="auto_hide_transparent">Amaga _la barra del tot quan s'amagui automàticament</s:String>
+    <s:String x:Key="version">Versió {0}</s:String>
+    <s:String x:Key="visit_on_github">Visiteu RetroBar a GitHub</s:String>
+    <s:String x:Key="taskbar_scale">Escala de la barra de tasques</s:String>
+    <s:String x:Key="taskbar_scale_1x">100%</s:String>
+    <s:String x:Key="taskbar_scale_2x">200%</s:String>
+    <s:String x:Key="taskbar_scale_current">Configuració actual: {0}%</s:String>
+    <s:String x:Key="debug_logging">Activa el registre de _depuració</s:String>
+    <s:String x:Key="check_for_updates">Compro_va si hi ha actualitzacions</s:String>
+    <s:String x:Key="show_exit_menu_item">Mostra l'opció de sortida en el menú conte_xtual</s:String>
+    <s:String x:Key="slide_taskbar_buttons">_Llista els botons de la barra de tasques</s:String>
```

Comment:
This refers to the sliding animation of the task buttons when tasks are opened. It also slides adjacent tasks when they are closed. It could perhaps be used for dragging as well. Currently, RetroBar only uses this animation when opening tasks.

Microsoft mistranslated both Spanish and Catalan, making it confusing: "Scroll buttons in the taskbar."
![image](https://github.com/user-attachments/assets/22ba3325-6e1e-4ed1-98ec-89f3f0e1554a)

In this case, it does not list, but enables the sliding animation:
```suggestion
    <s:String x:Key="slide_taskbar_buttons">Des_lissa els botons de la barra de tasques</s:String>
```